pyannote/database/loader.py

```diff
@@ -187,6 +187,47 @@ def __call__(self, file: ProtocolFile) -> Timeline:
         return self.loaded_[uri]
 
 
+class LabLoader:
+    """LAB loader
+
+    Parameters
+    ----------
+    path : str
+        Path to LAB file with ProtocolFile
```

Comment:
A loader is expected to be able to load annotations for **any** file, not just one particular file. Since LAB files do not contain `uri` information, `path` must contain a `{uri}` placeholder:

```suggestion
        Path to LAB file with mandatory {uri} placeholder.
```